mint-auditor/src/bin/main.rs

```diff
@@ -238,8 +256,8 @@ fn sync_loop(
     ledger_db: &LedgerDB,
     logger: &Logger,
 ) -> Result<(), Error> {
-    let conn = mint_auditor_db.get_conn()?;
     loop {
+        let conn = mint_auditor_db.get_conn()?;
```

Comment:
Correct, originally it was outside for saving connection/reconnecting time but I figured if we ever switch to a smaller db pool for whatever size (e.g. one that only has a single connection) this would potentially allow some other users to access the database in-between block syncs. We probably never will, and its all accessing a local file so it doesn't really matter.